trie/sync.go

```diff
@@ -473,7 +486,7 @@ func (s *Sync) children(req *nodeRequest, object node) ([]*nodeRequest, error) {
 				// and we hold the assumption that it's NOT legacy contract code.
 				var (
 					chash        = common.BytesToHash(node)
-					owner, inner = ResolvePath(child.path)
+					owner, inner = ResolvePath(child.path, false)
```

Comment:
Is explicit non-s.zk behavior intended?

Reply:
Yes, that's the source code on the mpt side.